### PR TITLE
Date, signature line changes

### PR DESCRIPTION
**Pull request by adamfast**

### CODE REVIEW

File: bylaws.md [hunk 1]
```diff
@@ -577,26 +577,52 @@ The corporation has issued no shares of stock, and has no shareholders.
...
+____________________
+Jen Myers, Director
+
+
+____________________
+Jeffrey Triplett, Director
+
+
+____________________
+Drew Winstel, Director
```
Comment (drewbrew) [inline bylaws.md]: Do you need Robert Andrew Winstel since Drew is not my first name?

Comment (KatherineMichel) [reply bylaws.md]: It's up to you. I usually put whatever is the most formal name I've been given.

Comment (drewbrew) [reply bylaws.md]: Let's go ahead and make the switch then.

Comment (KatherineMichel) [reply bylaws.md]: Sounds good. I went ahead and did it.

**Pull request merged**